tests/components/arcam_fmj/test_media_player.py

```diff
@@ -102,7 +101,8 @@ async def test_mute_volume(player, state, mute):
 
 async def test_name(player):
     """Test name."""
-    assert player.name == f"{MOCK_NAME} - Zone: 1"
+    data = await update(player)
```

Comment:
Side note: The tests should preferably be rewritten to not interact with the entity directly.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations